### PR TITLE
Bliss-1527 ruby_version and gem dependencies update

### PR DESCRIPTION
**Pull request by chr-wtag**
**[Update bliss_attachments gem to fix security vulnerabilities](https://fadendaten.atlassian.net/browse/BLISS-1527)**
`Ruby version` ~> `2.3.1`
`bundler version` ~> `1.17.3`

### CODE REVIEW

**Comment by nbi-wtag** [comment]
I approved

**Comment by far-wtag** [comment]
JIRA STory Link https://fadendaten.atlassian.net/browse/BLISS-1527

**Comment by nbi-wtag** [comment]
Approved

File: Gemfile.lock [hunk 1]
```diff
@@ -1,114 +1,155 @@
...
-    cocaine (0.5.8)
-      climate_control (>= 0.0.3, < 1.0)
-    diff-lcs (1.2.4)
-    erubis (2.7.0)
-    hike (1.2.3)
-    i18n (0.6.4)
-    journey (1.0.4)
-    jquery-rails (3.0.4)
-      railties (>= 3.0, < 5.0)
+    actioncable (5.2.3)
```
Comment (ddi-wtag) [inline Gemfile.lock]: Going from 3 to 5 sounds like a bit of a stretch, doesn't it?

How did you ensure that this change is working as expected?

Comment (chr-wtag) [reply Gemfile.lock]: We used the forked gem in our `local bliss` to update `vendor/cache` and performed testing in our machine. As we found no discrepancy in our local machine, we created an MR for [bliss](https://gitlab.com/fadendaten/bliss/merge_requests/2571) and deployed the branch to `bliss-staging`. So far, we tested, we did not find any compatibility issues in the staging environment.

**Pull request merged**